src/server/pps/server/s3g_sidecar.go

```diff
@@ -58,7 +58,7 @@ func (a *apiServer) ServeSidecarS3G() {
 			s.pachClient,
 			a.env.DB,
 			a.env.Listener,
-			a.env.Config.PPSPipelineName,
+			client.NewPipeline(a.env.Config.PPSPipelineName),
```

Comment:
Just wondering, we are gonna introduce `NewProjectPipeline` in a separate PR right?

Reply:
Yup, it’s in 8176, along with a lot of other bits.